Sources/ClientRuntime/Networking/Http/HttpContext.swift

```diff
@@ -1,370 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0.
-
-import struct Foundation.TimeInterval
```

Comment:
This type and its builder were moved to the new `Smithy` module and renamed to just `Context`.

The various keys and accessors have been split up & moved to the relevant modules.